incubator/jaeger/templates/hotrod-deploy.yaml

```diff
@@ -22,14 +22,12 @@ spec:
     spec:
       containers:
         - name: {{ template "jaeger.fullname" . }}-hotrod
-          image: {{ .Values.hotrod.image.repository }}:{{ .Values.hotrod.image.tag }}
+          image: {{ .Values.hotrod.image.repository }}:{{ .Values.tag }}
```

Comment:
It is better to keep the ability to set the tag independently

Reply:
@batazor There was a discussion about this a while back that the images should be in sync because of library compatibility. You can find the discussion here: https://github.com/helm/charts/pull/5645